PluginGenerator/PluginBuilder.cs

```diff
@@ -17,14 +17,34 @@ public class PluginBuilder
     {
         public const string SONARQUBE_API_VERSION = "4.5.2";
 
+        /// <summary>
+        /// The name of the package in which the plugin will be created
+        /// </summary>
+        private const string CorePluginPackageNameTemplate = "org.sonarqube.plugin.sdk.[PLUGIN_KEY]";
```

Comment:
Changed the default package namespace to refer to the SDK (rather than just being "myorg").